code/modules/antagonists/space_dragon/space_dragon.dm

```diff
@@ -10,6 +10,20 @@
 	var/list/datum/mind/carp = list()
 	/// The innate ability to summon rifts
 	var/datum/action/innate/summon_rift/rift_ability
+	/// Current time since the the last rift was activated.  If set to -1, does not increment.
+	var/riftTimer = 0
+	/// Maximum amount of time which can pass without a rift before Space Dragon despawns.
+	var/maxRiftTimer = 300
+	/// A list of all of the rifts created by Space Dragon.  Used for setting them all to infinite carp spawn when Space Dragon wins, and removing them when Space Dragon dies.
+	var/list/obj/structure/carp_rift/rift_list = list()
```

Comment:
These references don't seem to be handled anywhere if the antag datum is Destroyed, is it?